src/styles/theme.js

```diff
@@ -46,7 +46,7 @@ const layout = {
   maxWidth: "80rem",
   headerHeight: "3.6rem",
   footerHeight: "18.6rem",
-  sidebarWidth: "14.6rem",
+  sidebarWidth: "15.2rem",
```

Comment:
I made this a bit wider so that the entire `renature` logo was covered when the sidebar opens :)

Reply:
I like that for this lander, bu tsince this is going to be general we can't count on it

Reply:
ah yes that's a good point! I'll just reset it :)